pkg/ddc/alluxio/transform_optimization.go

```diff
@@ -243,10 +244,17 @@ func (e *AlluxioEngine) optimizeDefaultFuse(runtime *datav1alpha1.AlluxioRuntime
 	if len(runtime.Spec.Fuse.Args) > 0 {
 		value.Fuse.Args = runtime.Spec.Fuse.Args
 	} else {
+		readOnlyFuseOpts := "--fuse-opts=kernel_cache,ro,attr_timeout=7200,entry_timeout=7200"
```

Comment:
Compare to original parameters, I delete `nonempty` and `max_read=131072`.

- In libfuse 3, `nonempty` option has been removed. Mounting over non-empty directories is now always allowed in libfuse 3.

- In libfuse 3, the maximum size of read requests has to be specified both in struct `fuse_conn_info` and passed to fuse_session_new() using the `-o max_read=<n>` mount option.

Click [here](https://github.com/libfuse/libfuse/blob/master/ChangeLog.rst) to see ChangeLog of libfuse 3.

Reply:
I found a change log following the link you provide:
> For the time being, the maximum size of read requests has to be specified both there and passed to fuse_session_new() using the -o max_read=<n> mount option.

Does that mean for now we still need to reserve `-o max_read=<n>`?

Reply:
I also think it is better to set this configuration. But when I passed `max_read=131072`, the Alluxio Fuse goes wrong:

> fuse: error: init() and fuse_session_new() requested different maximum read size (0 vs 131072)

![fuse-error](https://github.com/fluid-cloudnative/fluid/assets/63225111/33b07a6d-596e-4d63-bbee-91223619fa51)

I think it is because that Alluxio Fuse doesn't set the same value to `max_read` field in struct `fuse_conn_info`. But I don't find related configuration in [Alluxio Configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html).

Reply:
My suggestion is not setting it in libfuse3, because it's deprecated and be adaptive according to the system implementation. https://man7.org/linux/man-pages/man8/mount.fuse3.8.html

Reply:
Okay, got it. Also, following up the [mount.fuse3 manual](https://man7.org/linux/man-pages/man8/mount.fuse3.8.html), it's now already deprecated so no need to set it again in `fuse_session_new()`. The PR now LGTM.